cmake/arbor-config.cmake.in

```diff
@@ -49,7 +49,7 @@ set(ARB_ARCH @ARB_ARCH@)
 set(ARB_MODCC_FLAGS @ARB_MODCC_FLAGS@)
 set(ARB_CXX @CMAKE_CXX_COMPILER@)
 set(ARB_CXX_FLAGS @CMAKE_CXX_FLAGS@)
-set(ARB_CXXOPT_ARCH @ARB_CXXOPT_ARCH@)
+set(ARB_CXX_FLAGS_TARGET @ARB_CXX_FLAGS_TARGET_FULL@)
```

Comment:
We don't have to use the same name in the exported variable that we use for the internal variable.
The internal variable (with the `_FULL` suffix) is a bit verbose, and was named differently from the `ARB_CXX_FLAGS_TARGET` because we don't want to edit a cached variable.

Reply:
The `build-catalogue` script fails if we don't use the same name.

Reply:
Fixed.